app.js

```diff
@@ -72,11 +78,25 @@ io.on('connection', (socket) => {
     // import map class
     // eslint-disable-next-line global-require
     const { GameMap } = require('./public/models/GameMap');
-    const mapFile = `./private/assets/maps/map${Utils.getRandomNumber(1, 6)}.json`;
+    //const mapFile = `./private/assets/maps/map${Utils.getRandomNumber(1, 6)}.json`;
+    const mapFile = `./private/assets/maps/map0.json`;
```

Comment:
also this should be commented and the line before uncommented

Reply:
damn again... sorry didnt notice :D

Reply:
fixed it